packages/ListBox/src/ListBox.js

```diff
@@ -177,27 +214,26 @@ export const propTypes = {
   ]),
 };
 
-ListBox.propTypes = {
-  ...propTypes,
-  children: PropTypes.node.isRequired,
-  height: PropTypes.number.isRequired,
-  placeholder: PropTypes.string.isRequired,
-};
-
 export const defaultProps = {
   children: null,
-  filter: null, // eslint-disable-line
-  footer: null, // eslint-disable-line
+  hasError: false,
   hasImplicitAll: false,
   height: 200,
+  id: null,
   isDisabled: false,
   isInline: false,
   isMulti: false,
   isOpen: null,
+  isReadOnly: false,
   onChange: () => {},
-  placeholder: null,
+  placeholder: "",
```

Comment:
why would you want to have a placeholder `= ""` ? wouldn't make sense if there is not placeholder there is not dom render? or what would be the benefit of not using null?